substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/ResourcesFeature.java

```diff
@@ -170,6 +178,14 @@ public void duringAnalysis(DuringAnalysisAccess access) {
         newResources.clear();
     }
 
+    private static Pattern[] compilePatterns(Set<String> patterns) {
```

Comment:
Another `mx eclipsformat` issue:
```patch
@@ -180,10 +180,10 @@
 
     private static Pattern[] compilePatterns(Set<String> patterns) {
         return patterns.stream()
-                .filter(s -> s.length() > 0)
-                .map(Pattern::compile)
-                .collect(Collectors.toList())
-                .toArray(new Pattern[]{});
+                        .filter(s -> s.length() > 0)
+                        .map(Pattern::compile)
+                        .collect(Collectors.toList())
+                        .toArray(new Pattern[]{});
     }
 
     @Override
```

Reply:
I am really not sure of the formatting. I use IntelliJ and I configured it to use the Eclipse code formatter as said in `compiler/docs/IDEs.md`...